File: app/src/debug/java/app/maw629/homerelay/share/ShareOverlayHostActivity.java
package app.maw629.homerelay.share;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

public class ShareOverlayHostActivity extends Activity {
    public static final int BACKGROUND_COLOR = 0xFF005E7A;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        View background = new View(this);
        background.setBackgroundColor(BACKGROUND_COLOR);
        setContentView(background);
    }
}

File: app/src/androidTest/java/app/maw629/homerelay/share/ShareOverlayHostActivity.kt
package app.maw629.homerelay.share

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.ui.Modifier
import androidx.compose.ui.graphics.Color

class ShareOverlayHostActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            Box(
                modifier = Modifier
                    .fillMaxSize()
                    .background(Color(BACKGROUND_COLOR))
            )
        }
    }

    companion object {
        const val BACKGROUND_COLOR = 0xFF005E7AL.toInt()
    }
}
